<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>网络检测</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>目标网址</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>接收</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>延时</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>速度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_recieved">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_duration">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_speed">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_detect_net">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>检测</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="target_ip_edit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>104</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="port_edit2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>104</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>231</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>数据传输配置</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>缓存配置</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>策略</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Fragmentation</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>大小</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="cache_size_edit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>211</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>compress config</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>policy</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Compress</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> algorithm</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Snappy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gzip</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>211</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>transfer config</string>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>protocol</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>tcp</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>udp</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Err Det</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Parity Check</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>DataTransfer</string>
    </property>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>progress</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>task</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_recieved_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_sended_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_speed_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_task_begin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>begin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_choose_file">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>ChooseFile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_detect_net_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="text_console">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>221</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>console</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="port_edit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_init">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>begin</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>563</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
